src/main/java/Task.java

```diff
@@ -0,0 +1,89 @@
+public class Task {
+
+    private static int numberOfTasks = 0;
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        setNumberOfTasks();
+        printNewTaskAddedMessage();
+    }
+
+    public Task() {
+    }
```

Comment:
Perhaps this constructor can be extracted since it's not called later and there is a constructor on top already?

Reply:
Hi Isabella! I added this because I didn't put "super(...)" as the first line of the constructors of deadline and event classes. This other constructor put here serves as a prevention for inducing errors. But tyvm for ur suggestion. :)